alacritty/src/display.rs

```diff
@@ -208,7 +210,9 @@ impl Display {
 
         // We should call `clear` when window is offscreen, so when `window.show()` happens it
         // would be with background color instead of uninitialized surface.
-        window.swap_buffers();
+        if event_loop.is_x11() {
```

Comment:
Checking for `is_x11()` will probably cause a build failure on non-linux/bsd systems. We need to put this behind a #[cfg(not(any(target_os = "macos", windows)))]`.

Reply:
It's a draft

Reply:
Just pointing it out since I noticed it anyways. :)